style.css

```diff
@@ -402,3 +433,193 @@ textarea:focus {
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    background-image: url("images/Header-llustration-desktop@2x.png");
+  }
+
+  #headline .headline-wrapper {
+    width: 920px;
+    height: 700px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 60px;
+  }
+
+  #headline .headline-wrapper h1 {
+    font-weight: 800;
+    font-size: 40px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    margin: 0;
+  }
+
+  #headline .headline-wrapper p {
+    margin: 10px 150px 24px 150px;
+    line-height: 28px;
+  }
+
+  #headline .headline-wrapper ul {
+    margin: 24px 0 24px 0;
+  }
+
+  #headline .headline-wrapper .btn-round {
+    margin-top: 24px;
+  }
+
+  /* ****Work Section**** */
+
+  #works .work-wrapper {
+    width: 1150px;
+    margin-top: 80px;
+  }
+
+  #works .work-wrapper .list-work {
+    display: grid;
+    justify-content: center;
+    grid-template-rows: repeat(2, 1fr);
+    grid-template-columns: repeat(2, 1fr);
+    column-gap: 24px;
+    row-gap: 24px;
+  }
+
+  #works .work-wrapper .list-work .work {
+    width: 369px;
+    height: 475px;
+  }
+
+  footer .footer-wrapper h2 {
+    width: 450px;
+    font-size: 30px;
+    margin-right: 24px;
+  }
+
+  #works .work-wrapper .list-work .work h2 {
+    padding-left: 16px;
+    padding-right: 16px;
+  }
+
+  /* ****About me**** */
+
+  #about-me {
+    width: 100%;
+    min-height: 350px;
+    align-items: flex-end;
+    background: url("images/illustration-about-me-desktop@2x.png");
+    background-position: top;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  #about-me .about-me-wrapper {
+    width: 80vw;
+  }
+
+  #about-me .about-me-wrapper .resume {
+    background: none;
+    margin-top: 80px;
+    padding-left: 120px;
+    padding-right: 120px;
+  }
+
+  #about-me .about-me-wrapper .list-skill {
+    flex-direction: row;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+  }
+
+  /* ****Footer Section**** */
+
+  footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background: url("images/illustration-2-contact-form-desktop@2x.png");
+    background-size: contain;
+    background-position: top;
+    background-repeat: no-repeat;
+  }
+
+  footer .footer-wrapper {
+    width: 80vw;
+    min-height: 500px;
+    display: flex;
+    flex-direction: row;
+    margin-top: 80px;
+  }
+
+  footer .footer-wrapper form {
+    width: 680px;
+  }
+
+  footer .footer-wrapper form label {
+    width: 100%;
+  }
+
+  footer .footer-wrapper form label input,
+  textarea {
+    box-sizing: border-box;
+    width: 100%;
+  }
+
+  footer .social-icons {
+    margin-top: 145px;
+  }
+
+  footer .my-email {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  #works .work-wrapper .list-work {
+    display: grid;
+    justify-content: center;
+    grid-template-rows: repeat(2, 1fr);
+    grid-template-columns: repeat(3, 1fr);
```

Comment:
- [ ] Great job on implementing the media query for large screen sizes with proper size and using a display property grid and flex to align your work section of the project :eagle:

Reply:
Hi @Debas-31 😄  
I don't understand this one. What is the change I need to do? 
Can you, please give me more detail about this when you will be free?

Reply:
Hi @Trast00, you did a great job on implementing the required core project requirement. Highlight what you did nothing to change in this section of the project :rocket: